identity/identity_suite_test.go

```diff
@@ -228,7 +228,6 @@ var _ = Describe("Identity", func() {
 		It("should throw a 400 with a descriptive message", func() {
 			var missingOrgIDJson = [...]string{
 				`{ "identity": {"account_number": "540155", "type": "User", "internal": {} } }`,
-				`{ "identity": {"account_number": "540155", "org_id": "1979710", "type": "User", "internal": {} } }`,
```

Comment:
Do you think it would be helpful to have two explicit passing cases where:

- `org_id` exists at the top level, but not in `internal`
- `org_id` does not exist at the top level, but does in `internal`

It looks like the current test setup for the success cases have it in both places. Definitely makes sense to remove this check, though!

Reply:
Yeah! let me add that